src/xcode/ENA/ENA/Source/Client/Security/iOS 12 Fallback/CryptoKitFallbacks.swift

```diff
@@ -346,3 +346,68 @@ extension String {
 		}
 	}
 }
+
+
+// MARK: - Digital Green Certificate
+
+struct DGCRSAKeypair {
+	
+	enum DGCRSAKeypairError: Error {
+		case keypairGeneration(String)	// Key generation failed
+		case publicKey // Unable to get public key representation
+	}
+	
+	let publicKey: SecKey
+	let privateKey: SecKey
+	
+	// The publicKey with added RSA Header and Base64 encoded
+	let publicKeyForBackend: String
+	
+	init?() throws {
+		let tag = Bundle.main.bundleIdentifier ?? "de.rki.coronawarnapp"
+		
+		let publicKeyAttr: [NSObject: NSObject] = [
+			kSecAttrIsPermanent: true as NSObject,
+			kSecAttrApplicationTag: ("\(tag)public".data(using: String.Encoding.utf8) ?? Data()) as NSObject,
```

Comment:
Just cosmetics but if we stick to the 'DGC'-theme here, a small hint what these keys are for would be nice.
```suggestion
			kSecAttrApplicationTag: ("\(tag).dgc.public".data(using: String.Encoding.utf8) ?? Data()) as NSObject,
```

Reply:
I would like to stick with the DGC since when we add the different key lengths the header needs to change depending on the key length etc.